src/aiperf/dataset/loader/accuracy_dataset_loader.py

```
@@ -27,13 +27,40 @@
+           what's being injected on their behalf.
+        3. ``None`` (no system prompt).
+    """
+    user_value = user_config.accuracy.system_prompt
+    if user_value is not None:
+        return user_value
+    benchmark = user_config.accuracy.benchmark
+    if benchmark is None:
+        return None
+    try:
+        meta = plugins.get_metadata(PluginType.ACCURACY_BENCHMARK, benchmark)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)